start-page/beta/index.html

```diff
@@ -2,39 +2,201 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 
-{% extends "_base-resp.html" %}
+{% extends "_new-base-resp.html" %}
 
 {% block channel %}beta{% endblock %}
-{% block title %}{{ _('Welcome to <span>Thunderbird Beta</span>') }}{% endblock %}
+{% block title %}{{ _('Welcome to <div><span class="tb-text">Thunderbird</span> <span class="beta-text">Beta</span></div>') }}{% endblock %}
```

Comment:
This doesn't need to be this complicated.
We can use something like this: `<div class="tb-text">Thunderbird <small class="beta-text">Beta</small></div>`